controllers/drcluster_controller.go

```diff
@@ -277,6 +272,34 @@ func filterDRClusterMCV(mcv *viewv1beta1.ManagedClusterView) []ctrl.Request {
 	}
 }
 
+func filterDRClusterSecret(reader client.Reader, secret *corev1.Secret) []ctrl.Request {
+	drcusters := &ramen.DRClusterList{}
+	if err := reader.List(context.TODO(), drcusters); err != nil {
+		return []reconcile.Request{}
```

Comment:
This swallow the error - it will be impossible to debug why we always fail here. I think in other case we log info or error level log. If we don't expect this to fail, logging an error will be helpful. If we find that this is common failure to we can silence the log later.

Reply:
Done